{"code": 10000, "msg": "服务器内部错误", "@timestamp": "2020-11-16 11:03:24.376825", "error": "django.db.utils.IntegrityError", "errormsg": "FOREIGN KEY constraint failed", "errorpath": ["File \"D:\\GitHub\\DRF-iView\\rbac\\views.py\", line 19, in post"], "method": "POST", "url": "/rbac/login", "user": null, "params": {}, "data": {"username": "kinson.liu", "password": "lqs4568349"}}
{"code": 11001, "msg": "登录失败，用户名或密码错误", "@timestamp": "2020-11-16 11:06:15.788350", "error": "libs.exceptions.LoginFailed", "errormsg": "登录失败，用户名或密码错误", "errorpath": ["File \"D:\\GitHub\\DRF-iView\\rbac\\views.py\", line 52, in post"], "method": "POST", "url": "/rbac/login", "user": null, "params": {}, "data": {"username": "zhijun.li", "password": "123@456"}}
{"code": 10000, "msg": "服务器内部错误", "@timestamp": "2020-11-16 14:23:36.759111", "error": "NameError", "errormsg": "name 'serializers' is not defined", "errorpath": ["File \"D:\\GitHub\\DRF-iView\\rbac\\account.py\", line 23, in post"], "method": "POST", "url": "/rbac/login", "user": null, "params": {}, "data": {"username": "kinson.liu", "password": "lqs4568349"}}
{"code": 10000, "msg": "服务器内部错误", "@timestamp": "2020-11-16 14:24:33.217662", "error": "NameError", "errormsg": "name 'TokenAuthtication' is not defined", "errorpath": ["File \"D:\\GitHub\\DRF-iView\\rbac\\account.py\", line 43, in post"], "method": "POST", "url": "/rbac/login", "user": null, "params": {}, "data": {"username": "kinson.liu", "password": "lqs4568349"}}
{"code": 10000, "msg": "服务器内部错误", "@timestamp": "2020-11-16 14:24:51.013087", "error": "NameError", "errormsg": "name 'Response' is not defined", "errorpath": ["File \"D:\\GitHub\\DRF-iView\\rbac\\account.py\", line 46, in post"], "method": "POST", "url": "/rbac/login", "user": null, "params": {}, "data": {"username": "kinson.liu", "password": "lqs4568349"}}
{"code": 10000, "msg": "服务器内部错误", "@timestamp": "2020-11-16 14:27:07.301038", "error": "django.db.utils.OperationalError", "errormsg": "no such table: auth_group", "errorpath": ["File \"D:\\GitHub\\DRF-iView\\libs\\__init__.py\", line 36, in get"], "method": "GET", "url": "/rbac/user/", "user": null, "params": {"page": "1", "page_size": "20"}, "data": {}}
{"code": 10001, "msg": "Token认证失败", "@timestamp": "2020-11-16 14:37:36.494644", "error": "libs.exceptions.AuthenticationFailed", "errormsg": "Token认证失败", "errorpath": ["File \"D:\\GitHub\\DRF-iView\\libs\\authtication.py\", line 25, in authenticate"], "method": "GET", "url": "/rbac/account", "user": null, "params": {}, "data": {}}
{"code": 10002, "msg": "您没有执行该操作的权限.", "@timestamp": "2020-11-16 14:38:17.288504", "error": "rest_framework.exceptions.PermissionDenied", "errormsg": "您没有执行该操作的权限。", "errorpath": [], "method": "POST", "url": "/rbac/organization/", "user": null, "params": {}, "data": {"id": null, "name": "111", "parent": 1}}
{"code": 10000, "msg": "服务器内部错误", "@timestamp": "2020-11-16 14:40:30.569046", "error": "KeyError", "errormsg": "'id'", "errorpath": ["File \"D:\\GitHub\\DRF-iView\\libs\\__init__.py\", line 58, in delete"], "method": "DELETE", "url": "/rbac/organization/", "user": null, "params": {"id": "7"}, "data": {}}
{"code": 10000, "msg": "服务器内部错误", "@timestamp": "2020-11-16 16:47:45.922854", "error": "django.db.utils.IntegrityError", "errormsg": "FOREIGN KEY constraint failed", "errorpath": ["File \"D:\\GitHub\\DRF-iView\\rbac\\views.py\", line 57, in post"], "method": "POST", "url": "/rbac/login", "user": null, "params": {}, "data": {"username": "kinson.liu", "password": "lqs4568349"}}
{"code": 10000, "msg": "服务器内部错误", "@timestamp": "2020-11-16 16:47:49.617971", "error": "django.db.utils.IntegrityError", "errormsg": "FOREIGN KEY constraint failed", "errorpath": ["File \"D:\\GitHub\\DRF-iView\\rbac\\views.py\", line 57, in post"], "method": "POST", "url": "/rbac/login", "user": null, "params": {}, "data": {"username": "kinson.liu", "password": "lqs4568349"}}
{"code": 10000, "msg": "服务器内部错误", "@timestamp": "2020-11-16 16:54:04.519265", "error": "django.db.utils.IntegrityError", "errormsg": "FOREIGN KEY constraint failed", "errorpath": ["File \"D:\\GitHub\\DRF-iView\\rbac\\views.py\", line 57, in post"], "method": "POST", "url": "/rbac/login", "user": null, "params": {}, "data": {"username": "kinson.liu", "password": "lqs4568349"}}
{"code": 11001, "msg": "登录失败，用户名或密码错误", "@timestamp": "2020-11-16 16:54:16.147164", "error": "libs.exceptions.LoginFailed", "errormsg": "登录失败，用户名或密码错误", "errorpath": ["File \"D:\\GitHub\\DRF-iView\\rbac\\views.py\", line 90, in post"], "method": "POST", "url": "/rbac/login", "user": null, "params": {}, "data": {"username": "zhijun.li", "password": "123@456"}}
{"code": 11001, "msg": "登录失败，用户名或密码错误", "@timestamp": "2020-11-16 16:57:31.093732", "error": "libs.exceptions.LoginFailed", "errormsg": "登录失败，用户名或密码错误", "errorpath": ["File \"D:\\GitHub\\DRF-iView\\rbac\\views.py\", line 90, in post"], "method": "POST", "url": "/rbac/login", "user": null, "params": {}, "data": {"username": "zhijun.li", "password": "123@456"}}
{"code": 10000, "msg": "服务器内部错误", "@timestamp": "2020-11-16 16:57:34.877611", "error": "django.db.utils.IntegrityError", "errormsg": "FOREIGN KEY constraint failed", "errorpath": ["File \"D:\\GitHub\\DRF-iView\\rbac\\views.py\", line 57, in post"], "method": "POST", "url": "/rbac/login", "user": null, "params": {}, "data": {"username": "kinson.liu", "password": "lqs4568349"}}
{"code": 10000, "msg": "服务器内部错误", "@timestamp": "2020-11-16 17:42:28.587026", "error": "django.db.utils.IntegrityError", "errormsg": "FOREIGN KEY constraint failed", "errorpath": ["File \"D:\\GitHub\\DRF-iView\\rbac\\views.py\", line 57, in post"], "method": "POST", "url": "/rbac/login", "user": null, "params": {}, "data": {"username": "kinson.liu", "password": "lqs4568349"}}
